Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'turbolinks', '~> 5'
 gem 'validate_url'
 gem 'uglifier', '>= 1.3.0'
+gem 'bcrypt'
```

Comment:
@SemyzSerhiy Gems should be alphabetically sorted within groups.
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Bundler/OrderedGems